kythe/data/simple_vnames.json

```diff
@@ -40,7 +40,7 @@
     }
   },
   {
-    "pattern": "/(.*)",
```

Comment:
The semantics of these patterns are equivalent to RE2::FullMatch patterns; adding an anchor here is working around a bug in the Rust indexer.